src/test/configure-feature.js

```diff
@@ -2,198 +2,223 @@ import { describe } from 'riteway';
-    expected: 1
-  });
-
-  assert({
-    given: 'the feature is disabled',
-    should: 'not render the Feature component',
-    actual: $('.feature').length,
-    expected: 0
-  });
-
-  assert({
-    given: 'the feature is disabled and there is no Fallback component',
-    should: 'not render the Fallback component',
-    actual: $('.fallback').length,
-    expected: 0
-  });
-
-  assert({
-    given: 'props',
-    should: 'pass through props',
-    actual: $('.prop-check').text(),
-    expected: propCheck
-  });
-});
-
-describe('configureFeature(Default)(FeatureName)(Feature)', async should => {
-  const { assert } = should();
-
-  const Default = createTestComponent('default');
-  const Feature = createTestComponent('feature');
-
-  const ConfiguredFeature = configureFeature(Default)('game')(Feature);
-
-  const FeatureWithContext = withContext(['lessons', 'game'])(
-    ConfiguredFeature
-  );
-
-  const propCheck = 'bacon and eggs';
-
-  const $ = dom.load(render(<FeatureWithContext propCheck={propCheck} />));
-
-  assert({
-    given: 'the feature is enabled',
-    should: 'no render the Default component',
-    actual: $('.default').length,
-    expected: 0
-  });
-
-  assert({
-    given: 'the feature is enabled',
-    should: 'render the Feature component',
-    actual: $('.feature').length,
-    expected: 1
-  });
-
-  assert({
-    given: 'the feature is enabled',
-    should: 'not render the Fallback component',
-    actual: $('.fallback').length,
-    expected: 0
-  });
-
-  assert({
-    given: 'props',
-    should: 'pass through props',
-    actual: $('.prop-check').text(),
-    expected: propCheck
-  });
+  {
+    const ActiveComponent = createTestComponent('active');
+    const InactiveComponent = createTestComponent('inactive');
+
+    const ConfiguredFeature = configureFeature(
+      InactiveComponent,
+      'game',
+      ActiveComponent
+    );
+
+    const features = ['game', 'bar', 'baz'];
+
+    const $ = dom.load(
+      render(
+        <Provider value={features}>
+          <ConfiguredFeature />
+        </Provider>
+      )
+    );
+
+    assert({
+      given: 'the feature is enabled',
+      should: 'not render the Inactive component',
+      actual: $('.inactive').length,
+      expected: 0
+    });
+
+    assert({
+      given: 'the feature is enabled',
+      should: 'render the Active component',
+      actual: $('.active').length,
+      expected: 1
+    });
+  }
+  {
+    const ActiveComponent = createTestComponent('active');
+    const InactiveComponent = createTestComponent('inactive');
+
+    const ConfiguredFeature = configureFeature(
+      InactiveComponent,
+      'game',
+      ActiveComponent
+    );
+
+    const features = [];
+
+    const $ = dom.load(
+      render(
+        <Provider value={features}>
+          <ConfiguredFeature />
+        </Provider>
+      )
+    );
+
+    assert({
+      given: 'the feature is not enabled and there is a Inactive component',
```

Comment:
I think we can remove this from all the givens throughout the file ` and there is a Inactive component'`, there is no logic based on whether is an Inactive component or not.